pom.xml

```diff
@@ -412,7 +412,7 @@
     <version.org.slf4j>1.6.1</version.org.slf4j>
     <version.dom4j>1.6.1</version.dom4j>
     <version.jfree>1.0.9</version.jfree>
-    <version.com.h2database>1.4.195</version.com.h2database>
+    <version.com.h2database>1.4.191</version.com.h2database>
```

Comment:
Is the H2 downgrade definitely required?

Reply:
Yes, it's needed. This H2 version is last running with JDK6.

Reply:
Thanks